src/mon/MonCommands.h

```diff
@@ -1071,6 +1071,12 @@ COMMAND("osd pool rmsnap "
 	"name=pool,type=CephPoolname "
 	"name=snap,type=CephString",
 	"remove snapshot <snap> from <pool>", "osd", "rw")
+COMMAND("osd pool rmsnap_again "
+	"name=pool,type=CephPoolname "
+	"name=lower_snapid_bound,type=CephInt,range=0 "
+	"name=upper_snapid_bound,type=CephInt,range=0 ",
+	"remove non-existing snapshot again <snap> from <pool>",
```

Comment:
Forces re-removal of already removed snapshots in the range ```[lower_snapid_bound, upper_snapid_bound)``` on pool ```<pool>``` in order to cause OSDs to re-trim them after fixing malformed snap_mapper entries.  See bug <bugid> for details.  See also OSD admin socket command fix_malformed_snapmapper_keys.